app/controllers/application_controller.rb

```diff
@@ -90,7 +90,17 @@ def self.session_key_prefix
   # This will rescue any un-handled exceptions
   rescue_from StandardError, :with => :error_handler
 
+  def self.handle_exceptions?
+    Thread.current[:application_controller_handles_exceptions] != false
```

Comment:
So, it's in this PR because it's been invaluable in the process of sorting through the many failures -- not because it is actually rails5 related.